stage2/01-sys-tweaks/files/rc.local

```diff
@@ -73,4 +73,12 @@ if [ ! -f /home/umbrel/statuses/service-configured ]; then
     fi
 fi
 
+# Display connection details on the TTY
+if command -v connection-details > /dev/null; then
```

Comment:
```sh
if [ -x "$(command -v "$1")" ]; then
   ...
fi
```

I like doing this, because `/dev/null` is not necessary, and it also checks if binary is actually executable.  Shorter and better.

Reply:
Ooh, I like this method, but it's worth noting this is actually not an equivalent test.

Checking if a command exists is not the same as checking for an executable file.

The original will work with shell builtin, functions, and binaries.
Your variant will only work with binaries.

Your variant is more appropriate for this use case since we are explicitly checking for a binary and not a builtin or function.

Reply:
I should probably also change `connection-details` to something more specific like `umbrel-connection-details` or maybe just `umbrel-details` to avoid false positives.